src/manifest.json

```diff
@@ -0,0 +1,69 @@
+{
+	"manifest_version": 3,
+	"name": "will be filled in by webpack from package.json",
+	"version": "will be filled in by webpack from package.json",
+	"description": "will be filled in by webpack from package.json",
+	"author": "will be filled in by webpack from package.json",
+	"browser_specific_settings": {
+		"gecko": {
+			"id": "{2f9f4409-05af-4441-82f0-13a3449dd1e6}",
+			"strict_min_version": "101.0"
+		}
+	},
+	"host_permissions": [
+		"https://ratemyprofessors.com/graphql",
+		"https://www.ratemyprofessors.com/graphql",
+		"https://raw.githubusercontent.com/carltonnorthern/nickname-and-diminutive-names-lookup/master/names.csv/",
+		"webRequest"
```

Comment:
![image](https://github.com/CalvinWu4/RIT-Rate-My-Professors-Extension/assets/26044296/ef234278-f1aa-41dd-8aba-0490797e2900)
this is causing this on the Chrome Web Store
is it safe to delete this?

Reply:
that may be another chrome vs firefox difference, if possible i would keep it in for firefox. 

there should be code that modifies the base manifest for whichever version is not the default - i believe this is at the top of the webpack config (you should see it manipulating the background/service worker entries to make the browsers happy. that would be the place to do it

Reply:
yeah it works w/o the webrequest in chrome